lib/osf-components/addon/components/meetings-footer/template.hbs

```diff
@@ -0,0 +1,55 @@
+<div class='row icon-bar m-v-lg'>
+    <div class='col-md-4 col-sm-4 text-center '>
+        <div class='p-v-md m-t-xl m-h-md'>
+            <i class='fa fa-eye fa-custom-7x' local-class='blue-icon icon-circle'></i>
```

Comment:
We have not been consistent, but in the majority of cases we've used https://github.com/martndemus/ember-font-awesome for Font Awesome icons. Baylee is working on upgrading to FA 5 and using https://github.com/FortAwesome/ember-fontawesome, however. For now, please just use `ember-font-awesome`.